include/cpp-statsd-client/UDPSender.hpp

```diff
@@ -116,21 +115,21 @@ class UDPSender final {
     //!@}
 
     //! Error message (optional string)
-    std::optional<std::string> m_errorMessage;
+    std::string m_errorMessage;
```

Comment:
on each call should we be resetting this to the empty string? it seems like previous behavior remembered whatever the last error was no matter how long ago it happened. is that desired?

Reply:
did this one last night too